docs/what-is-mindsdb.mdx

```diff
@@ -1,152 +1,113 @@
 ---
-title: MindsDB - Platform for Building AI
+title: Minds[DB] - AGI's Query Engine
```

Comment:
As we are not introducing/mentioning Minds in the MindsDB Docs -- shall we remove the square brackets around DB? Or is the idea to bring Minds into MindsDB docs?

Reply:
lets leave the brackets, this is intentional :)